README.md

```diff
@@ -1,6 +1,6 @@
 # :fire: Furnace
 
-> Melts down the [GOLD](https://github.com/govau/gold) components into a zip file.
+> Melts down the [GOLD ( Design System )](https://github.com/govau/designsystem) components into a zip file.
```

Comment:
remove the word `GOLD` and replace with `Design System` please :(

Reply:
😬 no